package.json

```diff
@@ -27,11 +27,11 @@
     "z-schema": "^3.16.1"
   },
   "peerDependencies": {
-    "fury": "3.0.0-beta.7"
+    "fury": "^3.0.0-beta.7"
```

Comment:
3.0.0-beta.8 can include semantically breaking changes so that would break 0.19.0 of fury-adapter-swagger. This should be a hard dependency.

Reply:
Fixed.